src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1239,7 +1239,7 @@ public void LangVersion()
             parsedArgs = DefaultParse(new[] { "/langversion:latest", "a.cs" }, _baseDirectory);
             parsedArgs.Errors.Verify();
             Assert.Equal(LanguageVersion.Latest, parsedArgs.ParseOptions.SpecifiedLanguageVersion);
-            Assert.Equal(defaultVersion, parsedArgs.ParseOptions.LanguageVersion);
+            Assert.Equal(LanguageVersion.CSharp7_1, parsedArgs.ParseOptions.LanguageVersion);
```

Comment:
Basically - putting literally 7_1 here was intentional.

---
In reply to: [101638922](https://github.com/dotnet/roslyn/pull/17169#discussion_r101638922) [](ancestors = 101638922,101625158)